pypeit/wavecalib.py

```diff
@@ -735,6 +763,47 @@ def run(self, skip_QA=False, debug=False):
 
         return self.wv_calib
 
+    def get_echelle_orders(self, pad=3):
+        """
```

Comment:
document a bit

Reply:
This method should not be here. Good catch.